src/java.base/unix/native/libnio/fs/UnixCopyFile.c

```diff
@@ -122,7 +155,8 @@ Java_sun_nio_fs_UnixCopyFile_transfer
         if (bytes_sent == -1) {
             if (errno == EINVAL || errno == ENOSYS) {
                 // Fall back to copying via user-space buffers
-                transfer(env, dst, src, cancel);
+                char buf[MIN_TRANSFER_SIZE];
```

Comment:
We can't have the fallback path using 16k of stack here.

Reply:
That code is likely never called.